libs/common/src/lib/generate-id.ts

```diff
@@ -1,5 +1,16 @@
-// Simple alphanumeric ID: https://stackoverflow.com/a/12502559/453959
-// This isn't a real GUID!
-export function generateId(): string {
-    return Math.random().toString(36).slice(2);
+/*
+Implementation was copied from Nano ID (https://github.com/ai/nanoid)
```

Comment:
Normally you must copy paste the original license to the copy pasted code

https://github.com/ai/nanoid/blob/main/LICENSE

Reply:
Done :)